Sources/Testing/Events/Recorder/Event.ConsoleOutputRecorder.swift

```diff
@@ -14,7 +14,7 @@ extension Event {
   ///
   /// The format of the output is not meant to be machine-readable and is
   /// subject to change.
```

Comment:
Consider mentioning via a "see also" comment the JUnitXML recorder type as a machine-readable alternative (since this explicitly describes ConsoleOutputRecorder as non-machine readable)

Reply:
Sure.

Reply:
Will do.